templating/overriding.rst

```diff
@@ -19,6 +19,11 @@ from the bundle to ``app/Resources/AcmeBlogBundle/views/Blog/index.html.twig``
 (the ``app/Resources/AcmeBlogBundle`` directory won't exist, so you'll need
 to create it). You're now free to customize the template.
 
+If you don't want to override the entire template but instead just want to 
+override a block, as of Symfony 3.4 you can now do with by extending the original
```

Comment:
Usually the "as of Symfony x.x, you can" are added via the versionadded blocks. Those blocks are removed with the next major I believe.

Reply:
Thanks @iltar